backend/exchanges/btcdirect.go

```diff
@@ -31,14 +36,31 @@ func GetBtcDirectSupportedRegions() []string {
 	return regions
 }
 
-// IsBtcDirectSupported is true if coin.Code and region are supported by BtcDirect.
-func IsBtcDirectSupported(coinCode coin.Code, region string) bool {
+// IsBtcDirectSupported is true if coin.Code is supported by BtcDirect.
+func IsBtcDirectSupported(coinCode coin.Code) bool {
```

Comment:
I changed this logic to make it matching the structure we use for Pocket and Moonpay (https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/4815ca377bc7f3e912830d7ae0195244286f9223/backend/exchanges/exchanges.go#L138-L139). To check that the region is supported we can use (as below) a simple `slice.contains()`


As an aside, I wonder if we would benefit by declaring these as interfaces instead, so every one will have methods with the exact same signature

Reply:
Good point, please do it! :D

It could also be used to uniform the endpoints that provide info for the exchange widgets loading, e.g., `getExchangeMoonpayBuyInfo` and `getExchangePocketURL`, but maybe it is something for a future PR.

Reply:
> Good point, please do it! :D

Do you mean refactoring as an interface? If that's ok for you, I'd create an issue and work on that on a separate PR :)

Reply:
(I actually had a look and it takes a bit more work than I thought. Still worth doing imho)